library-extension-rx/src/main/java/com/instacart/library/truetime/TrueTimeRx.java

```diff
@@ -12,22 +25,15 @@
 import io.reactivex.functions.Function;
 import io.reactivex.functions.Predicate;
 import io.reactivex.schedulers.Schedulers;
-import java.io.IOException;
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.Date;
-import java.util.List;
-import org.reactivestreams.Publisher;
 
 public class TrueTimeRx
       extends TrueTime {
 
     private static final TrueTimeRx RX_INSTANCE = new TrueTimeRx();
     private static final String TAG = TrueTimeRx.class.getSimpleName();
 
+    private Predicate<InetAddress> _addressFilter = null;
```

Comment:
why the underscore in the name? I don't think we use it anywhere else

Reply:
that's the way the fields in this library are marked